src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -46,17 +46,18 @@
         </IntProperty.DataSource>
     </IntProperty>
     <StringProperty Name="OutputName" Visible="False" />
-    <DynamicEnumProperty Name="OutputType" DisplayName="Output Type" EnumProvider="OutputTypeEnumProvider" Visible="False"/>
-    <EnumProperty Name="OutputTypeEx" DisplayName="Output Type" Visible="False">
-        <EnumValue Name="winexe" DisplayName="0" />
-        <EnumValue Name="exe" DisplayName="1" />
-        <EnumValue Name="library" DisplayName="2" />
-        <EnumValue Name="appcontainerexe" DisplayName="3" />
-        <EnumValue Name="winmdobj" DisplayName="4" />
-        <EnumProperty.DataSource>
+    <!-- This property is used to provide the value of OutputType through BrowseObject, which is used by Property pages-->
+    <StringProperty Name="OutputType" DisplayName="Output Type" Visible="False">
+        <StringProperty.DataSource>
+            <DataSource Persistence="ProjectFileWithInterception" PersistedName="OutputType" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext" />
+        </StringProperty.DataSource>
+    </StringProperty>
+    <!-- This property is used to provide the value of OutputTypeEx through BrowseObject, which is used by Property pages-->
+    <StringProperty Name="OutputTypeEx" DisplayName="Output Type Ex" Visible="False">
```

Comment:
We should use an IntProperty here - looks like that'll make it return an int to DTE